tests/test_bots.py

```diff
@@ -19,15 +21,26 @@ class TestBots(unittest.TestCase):
     def setUp(self, mock_login):
         self.ol = OpenLibrary()
 
-    def test__init__with_no_ol_arg(self):
-        OpenLibrary = Mock()
+    @patch('olclient.openlibrary.OpenLibrary')
+    def test__init__(self, mock_ol):
         bot = BaseBot()
-        assert OpenLibrary.assert_called_once()
+        assert bot.dry_run
+        assert bot.limit == 1
+        assert bot.logger.handlers
+        assert mock_ol.assert_called_once()  # FIXME not passing for some reason
```

Comment:
Using this pattern with `patch` usually behaves as expected, but for some reason I can't get this line, and similar lines, to pass. Anyone have any ideas?

Reply:
`assert mock_ol.assert_called_once()` --> `assert mock_ol.assert_called_once` ya dumb dumb